<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2ndWindow</string>
  </property>
  <widget class="QPushButton" name="purc_button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>구매</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>돌아가기</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>161</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>헬멧</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>팔꿈치 보호대</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>무릎 보호대</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>purc_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>next3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>before1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>next3()</slot>
  <slot>next1()</slot>
  <slot>before1()</slot>
 </slots>
</ui>
